roles/patroni/tasks/custom_wal_dir.yml

```diff
@@ -43,8 +43,13 @@
         dest: "{{ postgresql_wal_dir }}/"
       delegate_to: "{{ inventory_hostname }}"
 
-    - name: "Rename {{ 'pg_wal' if postgresql_version is version('10', '>=') else 'pg_xlog' }} to {{ 'pg_wal' if postgresql_version is version('10', '>=') else 'pg_xlog' }}_old"
-      ansible.builtin.command: mv {{ postgresql_data_dir }}/{{ 'pg_wal' if postgresql_version is version('10', '>=') else 'pg_xlog' }} {{ postgresql_data_dir }}/{{ 'pg_wal' if postgresql_version is version('10', '>=') else 'pg_xlog' }}_old
+    # 🔄 Determine base pg_wal_dir name
```

Comment:
It's interesting to see emojis used in code comments – they certainly add a unique touch! However, to ensure our code remains consistent, compatible, and as universally readable as possible, we've been maintaining a text-only policy in comments.

@ThomasSanson  If you have examples of such usage in major projects, could you please share them? It would help us understand your perspective better and facilitate a more informed discussion.

Reply:
Thank you,

Here are some resources that discuss the use of emojis in code comments or related areas :

1. [How Emoji Can Improve Your Code—Seriously - TJ VanToll](https://www.tjvantoll.com/2016/06/10/emoji-and-coding/): This article discusses how emojis can make commit messages more noticeable in large projects

2. [Complete list of GitHub markdown emoji markup - GitHub Gist](https://gist.github.com/rxaviers/7360908): This is a complete list of GitHub markdown emoji markup, which could be used in comments

3. [The Ultimate Manual to Github Emojis - Nira](https://nira.com/github-emojis/): This article explains how some GitHub users have created codes of emojis to indicate specific meanings in comments

4. [Let's talk about emojis - DEV Community](https://dev.to/fbnlsr/lets-talk-about-emojis-2788): This post discusses the use of emojis in code comments

5. [A First Look at Emoji Usage on GitHub: An Empirical Study - arXiv](https://arxiv.org/pdf/1812.04863): This empirical study looks at the usage of emojis in issues, pull requests, and comments on GitHub

6. [Emoji reactions - GitLab Docs](https://docs.gitlab.com/ee/user/award_emojis.html): This GitLab documentation page explains how emoji reactions can be used to give and receive feedback on individual comments

Reply:
I use https://gitmoji.dev/

Reply:
Just to clarify, I strive to write code that is primarily understandable by humans (developers, but not limited to them), rather than solely for the sake of the computer

Reply:
:beers: Ok. 

Thanks for the reply.